setup.sh

```diff
@@ -0,0 +1,63 @@
+set -e
+
+log_line() {
+  echo "$1"
+}
+
+install_jenv() {
+    if ! grep -q jenv "$1"; then
+      echo "export PATH=""$HOME"/.jenv/bin:"$PATH""" >> "$1"
+      echo "eval "$(jenv init -)"" >> "$1"
+      $("$2 $1")
+    fi
+}
+
+log_line '--- FormFlow Setup Script ---'
+
+if ! brew --version; then
+  curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh
+else
+  brew update
+fi
+
+log_line 'Installing brew packages'
+brew install openjdk@17 jenv gradle postgresql@14
```

Comment:
Do we need to install `openjdk@17` is it playing the same role as `temurin-17` in `jenv`? Or is it needed for something else?

Reply:
Good catch, that's a mistake I accidentally carried over to the script.. I can just choose Temurin and change the package.